style.css

```diff
@@ -737,16 +807,558 @@ ul {
   color: #172b4d;
 }
+
+#text {
+  display: flex;
+  height: 160px;
+  width: 305px;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  padding-left: 20px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #172d4d;
+}
+
+#contact-submit {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 48px;
+  width: 150px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  padding: 12px 16px 12px 16px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  margin-top: -20px;
+}
+
+#contact-submit:hover {
+  background-color: #6070ff;
+  font-size: 18px;
+  color: #fff;
+}
+
+#contact-submit:active {
+  background-color: #2230d2;
+  font-size: 18px;
+  color: #fff;
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+    background-color: #e5e5e5;
+  }
+
+  .logo {
+    font-size: 25px;
+    font-weight: 700;
+    line-height: 20px;
+    background-color: transparent;
+    transition: width 2s, height 2s, background-color 2s, rotate 2s;
+    rotate: 360deg;
+  }
+
+  .app-header {
+    display: flex;
+  }
+
+  .humbourger {
+    display: none;
+  }
+
+  .ambony {
+    width: auto;
+  }
+
+  .menu {
+    display: flex !important;
+    justify-content: flex-end;
+    align-items: center;
+    margin-right: 20px;
+  }
+
+  .menu-item {
+    margin-right: 20px;
+  }
+
+  .menu-item a {
+    display: flex;
+    font-size: 1rem;
+    color: #344563;
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  .header-ambony {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+    background-color: #fff;
+    background-image: url(media/Header\ bg.svg);
+    background-size: cover;
+    top: 2.875rem;
+    border-bottom-left-radius: 60px;
+    height: 44.88rem;
+  }
+
+  .header-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+  }
+
+  .narson {
+    width: 100%;
+  }
+
+  .detail {
+    height: 98px;
+    width: 920px;
+    left: 0;
+    top: 134px;
+  }
+
+  .connection {
+    margin-top: -1px;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #7f8cff;
+  }
+
+  /* work section */
+
+  .work {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    margin: 10px 0 0 28px;
+    margin-top: 5%;
+    border-radius: 8px;
+    background-color: #e5e5e5;
+  }
+
+  .card-container {
+    display: flex;
+    float: left;
+    flex-direction: row;
+    padding: 24px;
+    gap: 46px;
+    border: 1px solid #dfe1e6;
+    margin-top: 100px;
+    height: 60vh;
+    width: 100%;
+    border-radius: 24px;
+  }
```

Comment:
- [ ] Please let stick to the Figma design as much as possible as suggested by the previous reviewers. Therefore kindly make sure that your `card-container` are look like the figma design.

### Your design
![Screenshot from 2023-04-08 19-21-27](https://user-images.githubusercontent.com/91142432/230739467-45aa859e-cfbb-4030-9a32-9979598aa1ba.png)

### What it is supposed to look like
![Screenshot from 2023-04-06 10-25-14](https://user-images.githubusercontent.com/91142432/230739490-ef6473be-039e-484a-8392-d095c674aca0.png)